sim_pipeline/gg_lens_pop.py

```diff
@@ -72,13 +74,18 @@ def draw_population(self):
         # Estimate the number of lensing systems
         num_lenses = self._lens_galaxies.deflector_number()
         num_sources = self._source_galaxies.galaxies_number()
-        num_lens_systems = int(num_lenses * num_sources)
-        print(num_lens_systems)
-        for _ in range(100):
-            source = self._source_galaxies.draw_galaxy()
+        num_sources_tested_mean = (testarea * self.f_sky * num_sources)/12960000
+        print(num_sources_tested_mean)
+        print(np.int(num_lenses*num_sources_tested_mean))
+
+# Draw a population of galaxy-galaxy lenses within the area.
+        for _ in range(num_lenses):
```

Comment:
num_lenses seems not to be defined. I also don't see a test function for it

Reply:
![image](https://user-images.githubusercontent.com/122687108/235315381-99a4f05f-2ddb-4e0f-8b7d-f92677bb1baa.png)
        num_lenses = self._lens_galaxies.deflector_number()
I think it is defined, I will add test function about it.